traffic_ops/testing/api/v4/profiles_test.go

```diff
@@ -517,12 +517,31 @@ func GetTestProfiles(t *testing.T) {
 		}
 		profileID := resp.Response[0].ID
 
-		// TODO: figure out what the 'Parameter' field of a Profile is and how
```

Comment:
You can probably remove these commented lines now.

Reply:
Fixed.